.circleci/config.old.yml

```diff
@@ -447,6 +452,10 @@ workflows:
       - chromatic:
           requires:
             - examples
+  sandboxes:
+    when: << pipeline.parameters.run-js >>
```

Comment:
Just an example of only running a workflow if a certain type of file is modified in the PR. 

This new workflow might not make sense.